sunkit_image/conftest.py

```diff
@@ -183,3 +185,44 @@ def aia_171(request):
 def hmi_map():
     hmi_file = get_test_filepath("hmi_continuum_test_lowres_data.fits")
     return sunpy.map.Map(hmi_file)
+
+
+@pytest.fixture()
+def aia171_test_clipping():
+    return np.asarray([0.2, -0.3, -1.0001])
+
+
+@pytest.fixture()
+def aia171_test_map():
+    testpath = sunpy.data.test.rootdir
+    return sunpy.map.Map(Path(testpath) / "aia_171_level1.fits")
+
+
+@pytest.fixture()
+def aia171_test_shift():
+    return np.array([3, 5])
+
+
+@pytest.fixture()
+def aia171_test_map_layer(aia171_test_map):
+    return aia171_test_map.data.astype("float32")  # SciPy 1.4 requires at least 16-bit floats
+
+
+@pytest.fixture()
+def aia171_test_map_layer_shape(aia171_test_map_layer):
+    return aia171_test_map_layer.shape
```

Comment:
Shape is going to go away for maps, so we should use `.data.shape`.

Is there a case where we pass in the layer_shape but not the layer to the test function?

Reply:
This was used in one of the old unit test for testing the `match_template`. But we could just directly pass the `.data.shape` instead of having a fixture.